src/tasks/component-installers/devfile-workspace-operator-installer.ts

```diff
@@ -0,0 +1,246 @@
+/*********************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import Command from '@oclif/command'
+import * as execa from 'execa'
+import * as fs from 'fs-extra'
+import * as yaml from 'js-yaml'
+import * as Listr from 'listr'
+import * as path from 'path'
+
+import { CheHelper } from '../../api/che'
+import { KubeHelper } from '../../api/kube'
+import { isKubernetesPlatformFamily, isOpenshiftPlatformFamily } from '../../util'
+
+/**
+ * Handle setup of the dev workspace operator controller.
+ */
+export class DevWorkspaceTasks {
+  protected kubeHelper: KubeHelper
+  protected cheHelper: CheHelper
+  protected devWorkspaceServiceAccount = 'devworkspace-controller'
+  protected devWorkspaceRole = 'devworkspace-controller'
+
+  protected devWorkspaceEditWorkspaceClusterRole = 'edit-workspaces'
+  protected devWorkspaceViewWorkspaceClusterRole = 'view-workspaces'
+  protected devWorkspaceRoleBinding = 'devworkspace-controller'
+
+  protected namespace = 'devworkspace-controller'
+  protected deploymentName = 'devworkspace-controller'
+
+  constructor(private readonly flags: any) {
+    this.kubeHelper = new KubeHelper(flags)
+    this.cheHelper = new CheHelper(flags)
+  }
+
+  getTemplatePath() {
+    return path.join(this.flags.templates, 'devworkspace')
+
+  }
+
+  /**
+   * Returns list of tasks which setup dev-workspace.
+   */
+  getTasks(flags: any, command: Command): ReadonlyArray<Listr.ListrTask> {
```

Comment:
Do we have a plan to create delete tasks to uninstall dev workspace operator?

Reply:
Oh, I see comment from @tolusha https://github.com/che-incubator/chectl/pull/862#issuecomment-695907423

Reply:
I've added delete tasks